src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -835,9 +835,19 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
-                        ->booleanNode('disable_not_compromised_password')
-                            ->defaultFalse()
-                            ->info('Disable NotCompromisedPassword Validator: the value will always be valid.')
+                        ->arrayNode('not_compromised_password')
+                            ->addDefaultsIfNotSet()
+                            ->info('not-compromised password validator configuration')
+                            ->children()
+                                ->booleanNode('enabled')
+                                    ->defaultTrue()
+                                    ->info('Enable NotCompromisedPassword Validator. When disabled, the value will always be valid.')
```

Comment:
not sure this is clear enough, the cognitive overload of "not" "disabled" "always" is making it hard to understand in a snap to me.

Reply:
Also stumbled across the same issue, that's why I removed the second part in the first iteration (see PR review discussion). Please discuss the info label with @stof, then I'll integrate the concluded changes in my PR. 

Maybe it could be changed as follows:
```
When disabled, passwords will not be checked by the NotCompromisedPassword Validator, so the value will always be valid.
```